vminfo_parser/analyzer.py

```diff
@@ -320,13 +311,12 @@ def sort_by_disk_space_range(
         for column_to_drop in drop_columns:
             sorted_range_counts_by_environment.drop(column_to_drop, axis=1, inplace=True)
 
-        return column_padding, sorted_range_counts_by_environment
+        return sorted_range_counts_by_environment
```

Comment:
The annotations on the method signature, and the docstring need to be updated to match the new return

Reply:
fixed